src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,59 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+    
+    let newObjectValue = [];
+    let abc = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"];
+    for(let a=0; a<Object.values(obj).length; a++){
+        Object.values(obj)[a].push(Object.keys(obj)[a]);
```

Comment:
```js
for (let value of Object.values(obj) {}
```

Would work just fine.

```
 Object.values(obj)[a].push(Object.keys(obj)[a]);
```
This is very confusing. You should prolly break it down to multiple lines.